bundler.go

```diff
@@ -156,6 +159,9 @@ func New(c *Configuration) (b *Bundler, err error) {
 		return
 	}
 
+	// Hide LSUIElement (osx/macOS only)
+	b.hideLSUIElement = c.HideLSUIElement
```

Comment:
Could you set the `darwinAgentApp` at the very beginning of the function instead? In this block of code:

```go
// Init
	b = &Bundler{
		appName:      c.AppName,
		Client:       &http.Client{},
		environments: c.Environments,
}
```